mod/uservalidationbyemail/start.php

```diff
@@ -111,9 +111,10 @@ function uservalidationbyemail_check_manual_login(\Elgg\Event $event) {
 		if ($user->isEnabled() && $user->isValidated() !== false) {
 			return;
 		}
-		
-		if (elgg_get_plugin_user_setting('email_validated', $user->guid, 'uservalidationbyemail')) {
-			// email address already validated
+
+		$validated = elgg_get_plugin_user_setting('email_validated', $user->guid, 'uservalidationbyemail');
```

Comment:
you could also solve this with a default value... this way it is harder to read

Reply:
correct